src/System.Private.Uri/pkg/System.Private.Uri.pkgproj

```diff
@@ -12,8 +12,9 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <ProjectReference Include="win\System.Private.Uri.pkgproj" />
-    <ProjectReference Include="unix\System.Private.Uri.pkgproj" />
+    <ProjectReference Include="..\src\System.Private.Uri.builds" />
```

Comment:
Correct System.Private.Uri doesn't need to be un-split as it is part of the closure of StdLib.